client/src/components/TrailShowPage.js

```diff
@@ -1,12 +1,21 @@
 import React, { useState, useEffect } from "react"
+import { useParams, Link } from "react-router-dom"
+import ReviewForm from "./ReviewForm"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
 
 const TrailShowPage = (props) => {
+  const userStatus = props.user
+
   const [trail, setTrail] = useState({
     reviews: [],
   })
+  const { id } = useParams()
+
+  const [errors, setErrors] = useState([])
+  const [loginStatus, setLoginStatus] = useState(true)
```

Comment:
I would re organize this with the `useState` at the top

```js
  const [errors, setErrors] = useState([])
  const [loginStatus, setLoginStatus] = useState(true)
  const [trail, setTrail] = useState({
    reviews: [],
  })
  
  const userStatus = props.user
  const { id } = useParams()
```

Reply:
Order changed and looking good 👍